apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -161,6 +162,15 @@ defmodule Aecore.Chain.Worker do
     Logger.info(fn ->
       "Added block ##{new_block.header.height} with hash #{Base.encode16(new_block_hash)}, total tokens: #{inspect(total_tokens)}"
     end)
+
+    # Broadcasting notifications for new mined transaction(per account)
+    Enum.each(new_block.txs, fn(tx) -> 
+      if tx.data.from_acc != nil and tx.data.to_acc != nil do
+        Notify.broadcast_new_mined_transaction(Base.encode16(tx.data.from_acc), tx)
```

Comment:
the call should only be `Notify.broadcast_new_mined_transaction(tx)` inside the `broadcast_new_mined_transaction` you can do it twice and do the encoding